docs/man/keylime_registrar.8.rst

```diff
@@ -0,0 +1,109 @@
+=====================
+keylime_registrar.8
+=====================
+
+------------------------------
+Keylime registrar service
+------------------------------
+
+:Manual section: 8
+:Author: Keylime Developers
+:Date: September 2025
+
+NAME
+====
+
+keylime_registrar - Keylime registrar service for agent registration
+
+SYNOPSIS
+========
+
+**keylime_registrar**
+
+(Most operations require root privileges, use with sudo)
+
+DESCRIPTION
+===========
+
+The registrar is a long-running HTTP(S) service used by agents and the verifier during
```

Comment:
The verifier don't really uses the registrar HTTP service, but access it's database to get the agent information.